app/src/main/res/values-nn/strings.xml

```diff
@@ -358,4 +373,130 @@
+  <string name="comment_votes">Kommentar­røyster</string>
+  <string name="moderation">Moderering</string>
+  <string name="registrations">Registreringar</string>
+  <string name="applicant">Søkjar</string>
+  <string name="approve">Godkjenn</string>
+  <string name="deny">Avvis</string>
+  <string name="approved_by">Godkjend av</string>
+  <string name="denied_by">Avvist av</string>
+  <string name="answer">Svar</string>
+  <string name="deny_reason">Grunn for avvising</string>
+  <string name="applications_request_shorthand">Appar</string>
+  <string name="reports">Rapportar</string>
+  <string name="reporter">Rapportert av</string>
+  <string name="resolved_by">Løyst av</string>
+  <string name="unresolve">Opphev løysing</string>
+  <string name="unresolved_by">Løysing oppheva av</string>
+  <string name="reason">Grunngjeving</string>
+  <string name="creator">Opphavsperson</string>
+  <string name="message">Melding</string>
+  <string name="report_person">Rapporter %1$s</string>
+  <string name="block_person">Blokker %1$s</string>
+  <string name="undistinguish_comment">Fjern framheving av kommentar</string>
+  <string name="distinguish_comment">Framhev kommentar</string>
+  <string name="site_info_name">%1$s-info</string>
+  <string name="site_legal_info_name">Juridisk info for %1$s</string>
+  <string name="moderators">Moderatorar</string>
+  <string name="admins">Administratorar</string>
+  <string name="custom_thumbnail">Sjølvvalt miniatyrbilete</string>
+  <string name="alt_text">Alternativ tekst</string>
+  <string name="post_hidden">Innlegg gøymt</string>
+  <string name="post_unhidden">Innlegg ikkje gøymt</string>
+  <string name="hide_post">Gøym innlegg</string>
+  <string name="unhide_post">Fjern gøyming av innlegg</string>
+  <string name="show_upvotes">Vis opprøyster</string>
+  <string name="show_downvotes">Vis nedrøyster</string>
+  <string name="show_upvote_percentage">Vis opprøystings-%</string>
+  <string name="score">Poeng</string>
+  <string name="info">Info</string>
+  <string name="settings_about_baseline_profile">Baseline-profil</string>
+  <string name="installed">Installert</string>
+  <string name="not_installed">Ikkje installert</string>
+  <string name="unable_to_retrieve">Klarte ikkje henta</string>
+  <string name="blocks">Blokkering</string>
+  <string name="unblock">Fjern blokkering</string>
+  <string name="blocked_users">Blokkerte brukarar</string>
+  <string name="blocked_communities">Blokkerte fellesskap</string>
+  <string name="blocked_instances">Blokkerte instansar</string>
+  <string name="you_have_no_blocked_users">Du har ikkje nokon blokkerte brukarar</string>
+  <string name="you_have_no_blocked_communities">Du har ikkje nokon blokkerte fellesskap</string>
+  <string name="you_have_no_blocked_instances">Du har ikkje nokon blokkerte instansar</string>
+  <string name="torrent_help">Lemmy can use a technology called BitTorrent to view or stream media. [Meir informasjon](%1$s)</string>
```

Comment:
Btw this is partially still in English

Reply:
Good catch! This was a (partial) copy/paste error (a similar string is used in Lemmy). Now fixed.